suncet_processing_pipeline/make_level1.py

```diff
@@ -127,19 +125,21 @@ def __badpixel_removal(self, input_map, badpixel_path):
 
         return badpixel_corrected_map
 
-    def __cosmic_ray_removal(self, input_map, outlier_mask):
+    def __cosmic_ray_removal(self, input_map):
         """
         Interpolates over bad pixels in the input image using bicubic interpolation if the config file has
         cosmic_ray_removal set to True.
 
         Parameters:
             input_map (sunpy.map): Input 2D map (image) containing pixel values.
-            outlier_mask (numpy.ndarray): Boolean mask indicating bad pixels (True for bad pixels, False for good pixels).
 
         Returns:
             sunpy.map: Interpolated sunpy map with bad pixels replaced by interpolated values.
         """
         if self.config.cosmic_ray_removal:
+
+            outlier_mask = utilities.detect_outliers(input_map.data, 500)
+
```

Comment:
I'm not sure where this is coming from-- not me. Do you recognize this?

Reply:
That looks like a @MSKirk line

Reply:
I'm not sure how it got in this pull request-- did someone force push an old version over main? Is it bad if we let this sneak in with this PR? it seems harmless

Reply:
It looks like it was gobbled up from some commits that I did last week. I am not sure how it got aggregated into this PR. If it looks alright to you, we can include it. I think that is easier than trying to separate it back out again just to merge it.